mne/io/ctf/ctf.py

```diff
@@ -57,6 +57,10 @@ def read_raw_ctf(directory, system_clock='truncate', preload=False,
     Notes
     -----
     .. versionadded:: 0.11
+    .. note:: To read in the Polhemus digitization data (for example, from
+              a .pos file), include the file in the CTF directory. The
+              points will then automatically be read into the `mne.io.Raw`
+              instance via `mne.io.read_raw_ctf`.
```

Comment:
This is already the `Notes` section, putting the entire text in a `.. note::` seems... excessive :)
```suggestion

    To read in the Polhemus digitization data (for example, from
    a .pos file), include the file in the CTF directory. The
    points will then automatically be read into the `mne.io.Raw`
    instance via `mne.io.read_raw_ctf`.
```